quickwit/Cargo.toml

```diff
@@ -51,25 +51,25 @@ base64 = "0.21"
 byte-unit = { version = "4", default-features = false, features = ["serde"] }
 bytes = "1"
 chitchat = { git = "https://github.com/quickwit-oss/chitchat", rev = "4973853" }
-chrono = "0.4.23"
+chrono = { version = "0.4.24", default-features = false, features = ["clock", "std", "wasmbind"] }
 clap = { version = "=3.1", features = ["env"] }
 colored = "2.0.0"
 console-subscriber = "0.1.8"
 criterion = { version = "0.4", features = ["async_tokio"] }
 cron = "0.11.0"
-dialoguer = "0.10.3"
+dialoguer = "0.10.4"
```

Comment:
that's not useful...

It is a bit counterintuitive, but 
```dialoguer = "0.10.4"```

actually means any version greater than 0.10.4  and compatible with semver.

We can keep 0.10.4 and just delete refresh `Cargo.lock`.